PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# Description
+
+Add a summary of the change and which issue is fixed, including motivation, context, dependencies, etc. 
+
+## Type of Change
+
+Delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# Testing
+
+Description of tests ran to verify changes.
+
+# Checklist:
+
+- [ ] Code follows the style guidelines
+- [ ] Code has been commented in relevant places
+- [ ] Changes to necessary documentation
+- [ ] Added new tests for this new fix
+- [ ] New and existing unit tests pass locally with changes
```

Comment:
In your case you would also want to make sure the basic manual testing was performed after deploying the app.